src/structs/Webhook.ts

```diff
@@ -57,7 +59,33 @@ export class Webhook {
   }
 
   /**
-   * Middleware function to pass to express, sets req.vote to the payload
+   * Listening function for handling webhook requests
+   * @example
+   * app.post('/webhook', wh.listener((vote) => {
+   *   console.log(vote.user) // => 395526710101278721
+   * }))
+   * @param fn Vote handling function
+   * @returns An express request handler
+   */
+  public listener (fn: (payload: WebhookPayload, req?: Request, res?: Response, next?: NextFunction) => void | Promise<void>) {
+    return async (req: Request, res: Response, next: NextFunction) => {
+      const response = await this._parseRequest(req, res)
+      if (!response) return
+
+      res.sendStatus(204)
+
+      try {
+        await fn(response, req, res, next)
+      } catch (err) {
+        console.error(err)
```

Comment:
I wonder if we should be providing an error handling option in the webhook constructor for this instead since logging isn't meant to be the responsibility of this library

Reply:
Should it default to console.error?

Reply:
If there's no error handling function? Maybe